### PR TITLE
Cleanup ImageEditor

### PR DESCRIPTION
**Pull request by hughbe**
Extracted from (and tested in) #810

### CODE REVIEW

File: src/System.Windows.Forms.Design.Editors/src/System/Drawing/Design/ImageEditor.cs [hunk 1]
```diff
@@ -53,12 +53,11 @@ protected static string CreateFilterEntry(ImageEditor e)
         }
 
         /// <summary>
-        ///      Edits the given object value using the editor style provided by
-        ///      GetEditorStyle.  A service provider is provided so that any
-        ///      required editing services can be obtained.
+        /// Edits the given object value using the editor style provided by
+        /// GetEditorStyle.static A service provider is provided so that any
```
Comment (danmosemsft) [inline src/System.Windows.Forms.Design.Editors/src/System/Drawing/Design/ImageEditor.cs]: typo - static

**Comment by danmosemsft** [comment]
@merriemcgaw @zsd4yr the repo is configured to require a review from a member of dotnet/dotnet-winforms github team. Would you like to add me to that, or give me similar permissions?

**Comment by danmosemsft** [comment]
@hughbe if you can fix merge conflict and the typo, I can merge. ty!

**Comment by hughbe** [comment]
Rebased

**Comment by danmosemsft** [comment]
thanks @hughbe . You're the top contributor in this repo by lines changed now..

**Pull request merged**